Marlin/src/lcd/e3v2/proui/dwin_defines.h

```diff
@@ -78,3 +78,29 @@
 #define DEF_BEDPIDT TERN(PREHEAT_1_TEMP_BED, PREHEAT_1_TEMP_HOTEND, 60)
 #define DEF_PIDCYCLES 5
 
+/**
+ * ProUI internal feature flags
+ */
+#if ALL(SDCARD_SORT_ALPHA, SDSORT_GCODE)
+  #define PROUI_MEDIASORT
+#endif
+#if ENABLED(POWER_LOSS_RECOVERY)
+  #define PROUI_ITEM_PLR    // Tune > Power-loss Recovery
+#endif
+#if ENABLED(HAS_JUNCTION_DEVIATION)
+  #define PROUI_ITEM_JD     // Tune > Junction Deviation
+#endif
+#if ENABLED(LIN_ADVANCE)
+  #define PROUI_ITEM_ADVK   // Tune > Linear Advance
+#endif
+#if HAS_MESH
+  #define PROUI_MESH_EDIT   // Add a menu to edit mesh points
+#endif
+#if ANY(PROUI_PID_TUNE, MPC_AUTOTUNE) && DISABLED(DISABLE_TUNING_GRAPH)
```

Comment:
having DISABLED(DISABLE_TUNING_GRAPH) is kind of redundant... there is no other reference to this.
instead if a user wants to disable the PID graph, they can simply comment `PROUI_TUNING_GRAPH` out.
```suggestion
#if ANY(PROUI_PID_TUNE, MPC_AUTOTUNE)
```

Reply:
This is a hidden option for developer and troubleshooting use. It should be left in.